packages/client/lib/sync/fetcher/accountfetcher.ts

```diff
@@ -0,0 +1,98 @@
+import { Fetcher, FetcherOptions } from './fetcher'
+import { Account } from '@ethereumjs/util'
+// import { Chain } from '../../blockchain'
```

Comment:
can be removed?

Reply:
yea I don't know how this crept it's way in. will remove.